CHANGELOG.rst

```diff
@@ -39,10 +57,10 @@ Minor Changes
 - aws_acm - Update automatic retries to stabilize the integration tests.
 - aws_codecommit - Support updating the description
 - aws_kms - Adds the ``enable_key_rotation`` option to enable or disable automatically key rotation.
-- aws_kms: code refactor, some error messages updated
+- aws_kms - code refactor, some error messages updated'
```

Comment:
```suggestion
- aws_kms - code refactor, some error messages updated
```

Reply:
It's being generated by antsibull-changelog. I can manually remove it this release, and keep an eye if it pops back up again for 1.3.0 (or 1.2.1 if we have one).
changelogs/fragments/66037-aws_kms.yml in ansible/ansible was wrapped in single quotes so it's possible antsibull might not be expecting those?
cc @felixfontein

Reply:
@jillr this was manually added in #185 by yourself: https://github.com/ansible-collections/community.aws/pull/185/commits/7cc15bc4f1934960cee8f363ec3305086dd4a0ce#diff-7c54aa032dfbb9dff42ef068f965d4d4L86-R80 (I guess you simply forgot to remove the trailing apostrophe.)

(antsibull-changelog is using pyyaml to load and store YAML files and does not modify the contents of changelog fragments)

Reply:
Oh no I missed that, thanks!  :)